### ISSUE TITLE
Update `solr_helpers` gem

### ISSUE THREAD

**Issue by nichgray** [CDRH/lewisandclark#240]
Pagination landmarks need a unique label: this is popping an audit error on the search page. I think it should be fixed with https://github.com/CDRH/solr_helpers/pull/2 and https://github.com/CDRH/lewisandclark/pull/238/changes/b7ab714dade4977116f2671f8c44248cb343119d, but those changes should be checked (see https://github.com/CDRH/lewisandclark/issues/239).

**Comment by techgique** [comment]
Updated in https://github.com/CDRH/lewisandclark/commit/d2fa4d25cd04c3ce32a4edaa025ac04ed2fdf43f
